src/ARM.cpp

```diff
@@ -34,6 +35,43 @@
 using Platform::Log;
 using Platform::LogLevel;
 
+#ifdef GDBSTUB_ENABLED
+#define GDB_CHECK_A() do{\
```

Comment:
could you make these a member function of ARM which just has an empty body when GDBSTUB_ENABLED is not defined?